android/src/test/java/com/revenuecat/purchases/hybridcommon/CommonKtTests.kt

```diff
@@ -33,4 +44,199 @@ internal class CommonKtTests {
             setProxyURLString("this is not a url")
         }
     }
-}
\ No newline at end of file
+
+    private val mockApplicationContext = mockk<Application>(relaxed = true)
+    private val mockContext = mockk<Context>(relaxed = true)
+    private val mockPurchases = mockk<Purchases>()
+
+    @BeforeEach
+    fun setup() {
+        mockkObject(Purchases)
+        every {
+            Purchases.configure(
+                context = any(),
+                apiKey = any(),
+                appUserID = any(),
+                observerMode = any(),
+                service = any()
+            )
+        } returns mockPurchases
+        every { mockContext.applicationContext } returns mockApplicationContext
+    }
+
+    @Test
+    fun `calling logIn correctly passes call to Purchases`() {
+        val appUserID = "appUserID"
+
+        configure(
+            context = mockContext,
+            apiKey = "api_key",
+            appUserID = "appUserID",
+            observerMode = true,
+            platformInfo = PlatformInfo("flavor", "version")
+        )
+
+        every { Purchases.sharedInstance } returns mockPurchases
```

Comment:
i'm new to mockk, but couldn't you pull the `every { Purchases.sharedInstance } returns mockPurchases` to setup() if you call it in each test?